include/cppflow/tensor.h

```diff
@@ -167,14 +167,16 @@ namespace cppflow {
 
         // EXECUTE
         int n = 1;
-        TFE_TensorHandle* res[1];
+        TFE_TensorHandle* res[1] = { nullptr };
         TFE_Execute(op, res, &n, context::get_status());
         status_check(context::get_status());
         TFE_DeleteOp(op);
 
         tensor r;
         r.tf_tensor = { TFE_TensorHandleResolve(res[0], context::get_status()), TF_DeleteTensor};
```

Comment:
I think calling `tensor::tensor(TFE_TensorHandle* handle)` is better here. We can avoid setting the internal members and waiting for resolving the concrete tensor.

Reply:
`tensor::tensor(TFE_TensorHandle* handle)` has not set `this->tf_tensor`. Is it right?

Reply:
I don't think we need to set `tf_tensor` field here. For all eager operations, only the eager handle is used. If want to access the underlying data, we must call `TFE_TensorHandleResolve`. So, `tf_tensor` is really a temporary copy and does not hold the actual data. The data is in `tfe_handle`

Reply:
So is the member `tf_tensor` useless technically? Is it only a temporary member variable?

Reply:
I guess it was to provide some backward compatibility with tf v1, e.g. the saved model API, but it is used more or less like a temporary variable during construction.

Reply:
This proposal will be in a new PR.